packages/frontend/src/components/common/Filters/FilterInputs/DefaultFilterInputs.tsx

```diff
@@ -23,6 +23,73 @@ export type FilterInputsProps<T extends ConditionalRule> = {
     disabled?: boolean;
 };
 
+const getPlaceholderByFilterTypeAndOperator = (
+    filterType: FilterType,
+    filterOperator: FilterOperator,
+) => {
+    switch (filterType) {
+        case FilterType.NUMBER:
+            switch (filterOperator) {
+                case FilterOperator.EQUALS:
+                case FilterOperator.NOT_EQUALS:
+                case FilterOperator.LESS_THAN:
+                case FilterOperator.GREATER_THAN:
+                    return 'Enter Value(s)';
+                case FilterOperator.NULL:
+                case FilterOperator.NOT_NULL:
+                default:
+                    return '';
+            }
+        case FilterType.STRING:
+            switch (filterOperator) {
+                case FilterOperator.EQUALS:
+                case FilterOperator.NOT_EQUALS:
+                    return 'Start typing to filter results';
+                case FilterOperator.STARTS_WITH:
+                case FilterOperator.ENDS_WITH:
+                case FilterOperator.INCLUDE:
+                case FilterOperator.NOT_INCLUDE:
+                    return 'Enter value';
+                case FilterOperator.NULL:
+                case FilterOperator.NOT_NULL:
+                default:
+                    return '';
+            }
+        case FilterType.DATE:
+            switch (filterOperator) {
+                case FilterOperator.EQUALS:
+                case FilterOperator.NOT_EQUALS:
+                case FilterOperator.LESS_THAN:
+                case FilterOperator.LESS_THAN_OR_EQUAL:
+                case FilterOperator.GREATER_THAN:
+                case FilterOperator.GREATER_THAN_OR_EQUAL:
+                    return 'Select a date';
+                case FilterOperator.IN_THE_PAST:
+                case FilterOperator.NOT_IN_THE_PAST:
+                case FilterOperator.IN_THE_NEXT:
+                case FilterOperator.IN_THE_CURRENT:
+                    return 'Select a time period';
```

Comment:
Not needed

![image](https://github.com/lightdash/lightdash/assets/1983672/e2eac842-a95e-4a01-b042-9801bfe6c814)

Reply:
Will be needed when filters are allowed to be disabled in the UI, i.e. adding the filter, operator, but no value is set, yet

Reply:
exactly, not needed as you said @rephus , thanks! added a comment explaining as well